### PR TITLE
Improve file upload

### PR DESCRIPTION
**Pull request by Corepex**

### CODE REVIEW

**Review by markus-moser — APPROVED** [review]
Fine for a first improvement, we will work on some details in another PR

File: SECURITY.md [hunk 1]
```diff
@@ -1,22 +0,0 @@
-# Security Policy
```
Comment (vin0401) [inline SECURITY.md]: Was this removed by a reason?

File: assets/js/src/core/modules/execution-engine/index.ts [hunk 2]
```diff
@@ -17,13 +17,15 @@ import { type JobComponentRegistry } from './services/job-component-registry'
...
 import { NotificationJobContainer as DownloadJobContainer } from './jobs/download/notification-job-container'
+import { NotificationJobContainer as ZipUploadJobContainer } from './jobs/zip-upload/notification-job-container'
 
 export const executionEngineModule: AbstractModule = {
   onInit () {
     const jobComponentRegistry = container.get<JobComponentRegistry>(serviceIds['ExecutionEngine/JobComponentRegistry'])
 
     jobComponentRegistry.registerComponent('default', DefaultJobContainer)
     jobComponentRegistry.registerComponent('download', DownloadJobContainer)
+    jobComponentRegistry.registerComponent('unpack', ZipUploadJobContainer)
```
Comment (vin0401) [inline assets/js/src/core/modules/execution-engine/index.ts]: Can we rename the type to zip-upload. For me it would be more consistent.

**Review by vin0401 — CHANGES_REQUESTED** [review]
Just two small things, apart from that it looks pretty good already. :)

File: SECURITY.md (hunk 1 above)
Comment (Corepex) [reply SECURITY.md]: absolutely no idea - maybe a wrongly staged file in a merge or so ... but its back now https://github.com/pimcore/studio-ui-bundle/pull/549/commits/209fb36b65db1906beb7259cae2b309b74be5bc1

File: assets/js/src/core/modules/execution-engine/index.ts (hunk 2 above)
Comment (Corepex) [reply assets/js/src/core/modules/execution-engine/index.ts]: done with https://github.com/pimcore/studio-ui-bundle/pull/549/commits/f8356a8313ad21c7e4496d834db5eb15fa906ed4

**Review by vin0401 — APPROVED** [review]

**Comment by Corepex** [comment]
heureka :partying_face:

**Pull request merged**